.gitignore

```diff
@@ -22,3 +22,5 @@
 
 # Ignore .env files as they are personal
 /.env
+.editorconfig
```

Comment:
Hi, whats the reason to ignore those two files? Shouldn't the .editorconfig be shared with the team to ensure the same editor configuration?

Reply:
I think I copied that from somewhere else; I don't recall. I have no strong opinion about whether it is ignored or not. If folks were using different formatting standards they'd get a bunch of git conflicts over time, I think, though.